### DISCUSSION TITLE
Help with learning

### DISCUSSION THREAD

**Discussion by vacom13** [jax-ml/jax#8233] [Q&A] [answered]
Hi. I have been trying to get into ml and ai. Would love to contribute to this repository. Could someone please guide me on how to go about it? Basically pointing me towards specific resources I should learn.
P.S I am new to github so I am not sure if this place is the correct place to ask all this 😅.

**Comment by jakevdp** [comment, marked as answer]
Hi @vacom13 – that's a difficult question to answer without knowing more about your background and goals. Have you considered going through some kind of free online course to learn more about these topics?

For AI/ML, you might start with something like Andrew Ng's online Machine Learning course (I've known several students over the years who have found that very helpful). For developing your sofware skills, version control with git, open source contribution, etc. [Software Carpentry](https://software-carpentry.org/) might be a good place to start: they have excellent online written materials, as well as live courses & pre-recorded lectures.

Hope that helps, and best of luck!
